sql/ddl_api.sql

```diff
@@ -194,6 +194,12 @@ CREATE OR REPLACE PROCEDURE distributed_exec(
        transactional BOOLEAN = TRUE)
 AS '@MODULE_PATHNAME@', 'ts_distributed_exec' LANGUAGE C;
 
+-- Execute pg_create_restore_point() on each data node
+CREATE OR REPLACE FUNCTION create_distributed_restore_point(
```

Comment:
Is there a test, which calls this function and shows its result? I see it is called in the isolation test, but I don't see its output? Can you point or add such test?

Reply:
Result of this function is not deterministic or prone to be flaky, it is called like this  https://github.com/timescale/timescaledb/pull/2843/files#diff-760d95c35de4c932eb2777985ff2b367dc10a03a548931171becbe04ab81d76cR94 to be more stable in tests